osmosis-installer.py

```diff
@@ -589,13 +637,13 @@ def initEnvironment():
             if swapAns == "1":
                 swapNeeded = 32 - round(mem_gib)
                 print(bcolors.OKGREEN +"Setting up "+ str(swapNeeded)+ "GB swap file..."+ bcolors.ENDC)
-                subprocess.run(["sudo swapoff -a"], shell=True)
-                subprocess.run(["sudo fallocate -l " +str(swapNeeded)+"G /swapfile"], shell=True)
-                subprocess.run(["sudo chmod 600 /swapfile"], shell=True)
-                subprocess.run(["sudo mkswap /swapfile"], shell=True)
-                subprocess.run(["sudo swapon /swapfile"], shell=True)
-                subprocess.run(["sudo cp /etc/fstab /etc/fstab.bak"], shell=True)
-                subprocess.run(["echo '/swapfile none swap sw 0 0' | sudo tee -a /etc/fstab"], shell=True)
+                subprocess.run(["sudo swapoff -a"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True)
+                subprocess.run(["sudo fallocate -l " +str(swapNeeded)+"G /swapfile"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True)
+                subprocess.run(["sudo chmod 600 /swapfile"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True)

```

Comment:
Do we want to do this for stderr? I'm just thinking what if an error occurs and it's hard to tell where

Reply:
The only issue is as it currently stands, the installer clears the screen after this and moves to the next step. Maybe we should implement some kind of if statement to if stderr is not null that the user must acknowledge this. The only place this starts becoming difficult is some things in the sdk print out to stderror for zero reason. What do you think?

Reply:
Ahh, I see. I think we should implement your suggestion in a separate issue. For those unknown SDK stderr prints, we could just document in code the reasons for disabling stderr there